docs/content/docs/user-docs/install.md

```diff
@@ -47,7 +47,7 @@ Before installing a Helm chart, you must first make the Helm chart available on
 ```bash
 export HELM_EXPERIMENTAL_OCI=1
 export SERVICE=s3
-export RELEASE_VERSION=v0.0.1
+export RELEASE_VERSION=$(git -c 'versionsort.suffix=-' ls-remote --tags --sort='v:refname' https://github.com/aws-controllers-k8s/s3-controller.git | tail -n 1 | cut -d'/' -f3)
```

Comment:
I actually thought of proposing this in the original PR from @judyheflin :) However I decided against it because it would add `git` as a dependency for installation. I suppose a dependency on `git` isn't all that bad, but generally we're trying to keep the deps to a minimum and consider that installing folks may not be devs.

Reply:
I suppose so, sure.